### DISCUSSION TITLE
师傅您好，请问可以在生产环境中使用了吗？

### DISCUSSION THREAD

**Discussion by l-7-l** [mxsm/rocketmq-rust#1309] [Q&A] [answered]
> _太牛逼了师傅， 感谢您伟大的贡献 简直是传奇_

我在开发一个单体电商项目，初期不考虑高可用，请教下
      1. 该项目完成度如何 您建议在生产环境中使用吗？  
      2. 这里有一个kafka重制版 [iggy](https://github.com/iggy-rs/iggy)  您方便从性能和成熟度方面做下对比吗

> _顺祝身体健康 事业顺利 还有 天天开心~~_

**Comment by mxsm** [comment, marked as answer]
@l-7-l  暂时还不能使用在生成环境，项目整体还处于迭代的阶段。只是实现了基本的消息发送，Client的完成度算比较高。 如果感兴趣可以参与项目的迭代开发。 [iggy](https://github.com/iggy-rs/iggy)这个项目我倒是没有关注过，后续可以做一下对比。当前可能还是先把整体的功能赶上Java版本的进度。后续会进行优化，目标是让比Java版本的性能更加高

**Reply by l-7-l** [reply]
@mxsm 打扰下师傅  可以先使用这个rust版本的client-sdk 先对接java版本的rocket-mq server吗？ 您方便乐观估计下大概需要多久能可用 ？ 多久能高可用吗？ 

> _师傅牛逼 师傅666， 最后两个问题有点无耻 相信我 如果采用rocket-mq路线的话 等我活下来必拉你或赞助你_

> _顺祝事业顺利 平安健康_
